sources/packages/api/src/route-controllers/education-program/education-program.controller.ts

```diff
@@ -280,4 +307,62 @@ export class EducationProgramController {
       programDeclaration: program.programDeclaration,
     };
   }
+
+  /**
+   * Education Program Details for ministry users
+   * @param programId program id
+   * @returns programs details.
+   * */
+  @AllowAuthorizedParty(AuthorizedParties.aest)
+  @Groups(UserGroups.AESTUser)
+  @Get(":programId/aest")
+  async getProgramForAEST(
+    @Param("programId") programId: number,
+  ): Promise<EducationProgramData> {
+    const program = await this.programService.getEducationProgramDetails(
+      programId,
+    );
+
+    if (!program) {
+      throw new NotFoundException("Not able to find the requested program.");
+    }
+
+    return transformToEducationProgramData(program);
+  }
+
+  /**
+   * Get all programs of an institution with pagination
+   * for ministry users
+   * @param institutionId id of the institution.
+   * @param pageSize is the number of rows shown in the table
+   * @param page is the number of rows that is skipped/offset from the total list.
+   * For example page 2 the skip would be 10 when we select 10 rows per page.
+   * @param sortColumn the sorting column.
+   * @param sortOrder sorting order.
+   * @param searchProgramName Search the program name in the query
+   * @returns ProgramsSummaryPaginated.
+   */
+  @AllowAuthorizedParty(AuthorizedParties.aest)
+  @Groups(UserGroups.AESTUser)
+  @Get("institution/:institutionId")
```

Comment:
API URL can have aest in it specifying this is for Ministry. Also it will help to avoid potential router conflicts.